src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method 'getStatistic' is doing too much: reading from a file, processing data, and writing to another file. Consider breaking this method into smaller methods, each responsible for a single task.